__tests__/package.json.test.js

```diff
@@ -1,5 +1,5 @@
 describe('package.json', () => {
```

Comment:
IIRC tests are found if they're in the `__tests__` directory or if they end in `.test.js`, We tend to just do the latter.

Can you rename this to `tests/package.json.test.js`